### DISCUSSION TITLE
Implement personal arena rating

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#5362] [Suggestions]
Hello Chromiecraft team,

I wanted to suggest a new feature that I believe would greatly benefit the server's PvP community. Currently, the server uses a team arena rating system where two players form a team and gain or lose rating based on wins or losses. However, I believe that implementing a personal arena rating system would encourage more players to participate in arena matches and create a more dynamic and active PvP environment.

Under a personal arena rating system, each player would have an individual rating attached to their character. This would allow them to freely team up with any other player and participate in matches whenever they want, without having to wait for a specific teammate or schedule. This would be particularly beneficial for a low-population RP server like Chromiecraft, where PvP activity can be scarce.

To view their personal rating, players could use a command that displays their current rating. In addition, the server could implement a custom vendor that sells tokens that would allow players to purchase gear requiring a certain rating.

I believe that implementing a personal arena rating system would greatly enhance the PvP experience on Chromiecraft and encourage more players to participate in arena matches. Thank you for considering this suggestion, and please let me know if you have any questions or feedback.

**Comment by lamsxx** [comment]
If you're looking for a good idea to revitalize PVP, I suggest implementing 1 versus 1 fights for players who can't find a gaming partner or solo matches in 2 versus 2 or 3 versus 3. Since most players don't play PVP, it's important to make this part of the game as interesting as PVE

Biti.

**Reply by DuvanRCuero** [reply]
Arenas in wow aren't ment to be 1v1 for a reason

**Comment by fabi1010** [comment]
Solo queue would be a great addition as well. 

So Teams are randomly picked into 3v3 teams and you get a solo queue rating as well like normal arena. :P

**Comment by Helias** [comment]
>  suggest implementing 1 versus 1
> Solo queue would be a great addition as well.

We finally released them ;-)

Release article and video references
[youtube.com/watch?v=P7hYa2rwWqo](https://www.youtube.com/watch?v=P7hYa2rwWqo)
[chromiecraft.com/en/pvp-revamp-arena-s3](https://www.chromiecraft.com/en/pvp-revamp-arena-s3/)

> Arenas in wow aren't ment to be 1v1 for a reason

Partially agree, indeed we lowered a lot the ArenaPoints reward for it

> players could use a command that displays their current rating

Through the 1v1 and 3v3 soloQ npcs you can see the statistics but I opened these issues to implement a command to see the stats
https://github.com/azerothcore/mod-arena-3v3-solo-queue/issues/23
https://github.com/azerothcore/mod-1v1-arena/issues/61
Moreover, all the stats are present in the ArenaStats web tool https://www.chromiecraft.com/en/arena-stats/
